UniversalAlgebra/Homomorphisms/Basic.lagda

```diff
@@ -151,8 +155,10 @@ mon-to-hom : (𝑨 : Algebra 𝓤 𝑆){𝑩 : Algebra 𝓦 𝑆} → mon 𝑨 
 mon-to-hom 𝑨 ϕ = ∣ ϕ ∣ , fst ∥ ϕ ∥
 
 epi-to-hom : {𝑨 : Algebra 𝓤 𝑆}(𝑩 : Algebra 𝓦 𝑆) → epi 𝑨 𝑩 → hom 𝑨 𝑩
-epi-to-hom _ ϕ = ∣ ϕ ∣ , fst ∥ ϕ ∥
+epi-to-hom 𝑩 ϕ = ∣ ϕ ∣ , fst ∥ ϕ ∥
```

Comment:
I would have moved this in the opposite direction, i.e. made the argument implicit, as it is not used?  Same in `epi-is-surj` below?

Reply:
I can't remember why I switch from underscore to naming the argument.  Probably just to make it a bit clearer what the first explicit argument is here. However, I don't want to make B implicit because it is almost always necessary to specify the codomain when applying epi-to-hom.